packages/core-data/src/selectors.ts

```diff
@@ -281,22 +289,23 @@ export interface GetEntityRecord {
 			| Partial< ET.EntityRecord< any > >
 	>(
 		state: State,
-		kind: string,
-		name: string,
-		key: EntityRecordKey,
-		query?: GetRecordsHttpQuery
+		kind: EntityRecordArgs[ 0 ],
+		name: EntityRecordArgs[ 1 ],
+		key: EntityRecordArgs[ 2 ],
+		query?: EntityRecordArgs[ 3 ]
```

Comment:
@jsnajdr I'm not sure how "legit" this is but it seemed to make sense to reuse the `tuple` args.

Reply:
✅ Reverted to inlining